core/src/main/resources/lib/layout/breadcrumbs.js

```diff
@@ -28,7 +28,7 @@ var breadcrumbs = (function() {
     }
 
     Event.observe(window,"load",function(){
-      menu = new YAHOO.widget.Menu("breadcrumb-menu", {position:"dynamic", hidedelay:1000, zIndex:2001});
+      menu = new YAHOO.widget.Menu("breadcrumb-menu", {position:"dynamic", hidedelay:1000, zIndex:2001, scrollincrement: 2});
```

Comment:
Is this a YahooUI property?

Reply:
Yes:

https://github.com/jenkinsci/jenkins/blob/cc3d24c67b62cf6baa37e4e350889ee86c5ccce8/war/src/main/webapp/scripts/yui/menu/menu-debug.js#L6345-L6362

Reply:
YahooUI never ceases to surprise me